configure.ac

```diff
@@ -57,6 +57,12 @@ PKG_CHECK_MODULES(WEBSOCKETS, [libwebsockets >= 2.4.2],
 AC_SUBST(WEBSOCKETS_CFLAGS)
 AC_SUBST(WEBSOCKETS_LIBS)
 
+PKG_CHECK_MODULES(RABBITMQ, [librabbitmq >= 0.10.0],
```

Comment:
Use the 0.9.0 version. It is a release version found on https://github.com/alanxz/rabbitmq-c.

Reply:
Done